core/src/components/button/button.ios.scss

```diff
@@ -149,7 +149,7 @@
 }
 
 ::slotted(ion-icon[slot="icon-only"]) {
-  font-size: 1.125em;
+  font-size: dynamic-font-clamp(0.844, $button-ios-baseline-font-size, 2.53, 1.125em);
```

Comment:
As discussed internally, we should keep the default font sizes the same and just change the max/min values. For example, this line would become

```css
font-size: dynamic-font-clamp(0.844, 1.125em, 2.53);
```

Reply:
In order to get to `em`, I would have to pass a fourth value: `dynamic-font-clamp(0.844, 1.125em, 2.53, 1em);`. Otherwise, it coverts it to `rem` as the [default](https://github.com/ionic-team/ionic-framework/blob/a313ee85edf637017a3d3d4bd5ce415bc9c0e72a/core/src/themes/ionic.functions.font.scss#L4-L5).

Reply:
Yes good point! I forgot we want to keep `em`.